media/stylus/includes/vars.styl

```diff
@@ -173,7 +173,7 @@ $selector-heading-font-fallback = 'html[data-ffo-opensanslight="false"]:not(.no-
 paths
 ====================================================================== */
 $path-to-assets = '/media/';
-$path-to-images = $path-to-assets + 'img/';
+$path-to-images = '../../static/img/';
 $logo-sprite-url = $path-to-images + 'logo_sprite.svg?2014-01';
```

Comment:
No problem at all. It's a great point. I made a note to address that as part of the Stylus move.